src/main/java/com/faforever/client/mod/ModManagerController.java

```diff
@@ -86,6 +97,22 @@ public void initialize() {
     loadActivatedMods();
 
     modListView.scrollTo(modListView.getSelectionModel().getSelectedItem());
+
+    modSearchTextField.textProperty().addListener(new ChangeListener<String>() {
+      @Override
+      public void changed(ObservableValue<? extends String> observable, String oldValue, String newValue) {
+        if (newValue.isEmpty()) {
+          modVersionFilteredList.setPredicate(null);
```

Comment:
We would want to still preserve the ui or sim filter when the field is empty

Reply:
fixed